cmd/kube-scheduler/app/server.go

```diff
@@ -243,7 +254,15 @@ func newHealthzHandler(config *componentconfig.KubeSchedulerConfiguration, separ
 	healthz.InstallHandler(pathRecorderMux)
 	if !separateMetrics {
 		configz.InstallHandler(pathRecorderMux)
-		pathRecorderMux.Handle("/metrics", prometheus.Handler())
+		defaultMetricsHandler := prometheus.Handler().ServeHTTP
```

Comment:
Instead of duplicating the same code as above, could you move this part into a separate function?
Like here - https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/routes/metrics.go#L44

Reply:
done